packages/stateManager/index.ts

```diff
@@ -567,16 +570,23 @@ class KeyCardManager extends PublicObjectManager<KeyCard> {
       //storing WA and KC pair
       const kc = event.newKeyCard;
       const userWalletAddr = bytesToHex(kc.userWalletAddr!);
-      const cardPublicKey = bytesToHex(kc.cardPublicKey!);
-      await this.store.put(cardPublicKey, userWalletAddr);
-      this.emit("newKeyCard", cardPublicKey);
+      //Returns hex encoding of address.
+      const addressFromPubKey = Address.fromPublicKey(
+        kc.cardPublicKey,
+      ).toString();
+      await this.store.put(addressFromPubKey, userWalletAddr);
+      this.emit("newKeyCard", addressFromPubKey);
       return;
     }
   }
 
   get(key: `0x${string}`) {
     return this.store.get(key);
   }
+  addRelayAddress(key: `0x${string}`) {
```

Comment:
ah.. just to make the naming a bit clearer. I thought we might want to different functions (e.g. `addRelay` and `addKeyCard`) since we have two different paths (from the contract registries and from the event stream/account messages). But if the stateManager only persists the addresses, let's just call it `addAddress`.

put differently, keyCards are not relays 🙃